.github/workflows/publish-image.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
         with:
           context: .
+          platforms: linux/amd64,linux/arm64
```

Comment:
I'm pretty certain this will also require

```
 -
        name: Set up QEMU
        uses: docker/setup-qemu-action@v2
-
        name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2
```
Can you please also do a test run of the pipeline in your repository and link to the successful result?

Reply:
Added the parts I missed, here's the successful build: https://github.com/lryanuk/chiadog/actions/runs/2883959326

Ran the container on a Raspberry Pi4, seems to be working as expected

Reply:
That's great, thanks! 🙌